ChangeLog.md

```diff
@@ -1,3 +1,182 @@
+# 1.3.2 -> 1.4.0
+
+### General
+* Implemented High Score screen
+* Implemented tracking of completed campaigns
+* "Secret" Heroes 3 campaigns now require completion of prerequisite campaigns first
+* Completing a campaign will now return player to campaign selection window instead of main menu
+* Game will now play video on winning or losing a game
+* Game will now correctly check for mod compatibility when loading saved games
+* Game client will no longer load conflicting mods if player have both of them enabled
+* If some mods fail to load due to missing dependencies or conflicts, game client will display message on opening main menu
+* Game will no longer crash on loading save with different mod versions and will show error message instead
+* Saved games are now 2-3 time smaller than before
+* Added Vietnamese translation
+* Failure to connect to a MP game will now show proper error message
+* Added VSync support
+* Implemented tutorial
+* Slightly reduced server connection time
```

Comment:
Does that reference a different change than https://github.com/vcmi/vcmi/pull/2959 ?

Because that had to be reverted.

Reply:
Removed